dual_net_edge_tpu.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This file contains an implementation of dual_net.py for Google's EdgeTPU.
+It can only be used for inference and requires a specially quantized and
+compiled model file. For more information see https://coral.withgoogle.com
+"""
+
+import numpy as np
+import features as features_lib
+from edgetpu.basic.basic_engine import BasicEngine
+
+class DualNetworkEdgeTpu(object):
+  def __init__(self, save_file, board_size=19):
```

Comment:
Please use the `go.N` global variable for the board size, which is controlled by the `BOARD_SIZE` environment variable (and defaults to 19):
https://github.com/tensorflow/minigo/blob/6694df26917ca54173878ff1eeaa97c5e40d9265/go.py#L31